cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/ClusterModel.java

```diff
@@ -1242,23 +1238,6 @@ public BrokerStats brokerStats(KafkaCruiseControlConfig config) {
     return brokerStats;
   }
 
-  /**
-   * The variance of the derived resources.
-   * @return A non-null array where the ith index is the variance of RawAndDerivedResource.ordinal().
-   */
-  public double[] variance() {
-    RawAndDerivedResource[] resources = RawAndDerivedResource.values();
-    double[][] utilization = utilizationMatrix();
-
-    double[] variance = new double[resources.length];
-
-    for (int resourceIndex = 0; resourceIndex < resources.length; resourceIndex++) {
-      Variance varianceCalculator = new Variance();
-      variance[resourceIndex] = varianceCalculator.evaluate(utilization[resourceIndex]);
-    }
-    return variance;
-  }
-
```

Comment:
Why do we remove this method? I did a code search and I guess it is not related to this patch but because it is not used anywhere, right?

Reply:
This refactoring is irrelevant to this patch. To keep this PR self-contained, I will revert this change, and create a new refactoring PR for the meta-changes.